smoke-tests/apps/JavaUtilLogging/src/main/java/com/microsoft/applicationinsights/smoketestapp/JavaUtilLoggingDisabledServlet.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.smoketestapp;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/testDisabled")
+public class JavaUtilLoggingDisabledServlet extends HttpServlet {
+
+  private static final Logger logger = Logger.getLogger("smoketestapp");
+
+  protected void doGet(HttpServletRequest request, HttpServletResponse response) {
+    logger.log(Level.WARNING, "this message will get suppressed.");
+  }
+}
```

Comment:
I don't think we need a separate Servlet for these tests, can just hit the same servlet that the normal test does (this is how other `*DisabledTest` smoke tests work)

Reply:
i followed the existing setup.. one test per servlet. i think we can clean up later for all

Reply:
i noticed those are spring boot apps. not servlet